### DISCUSSION TITLE
Event-format-V2 ?

### DISCUSSION THREAD

**Discussion by JanneSaare** [Thomas55555/aioautomower#541] [General]
Hi
I’ve got an email from Husquvarna stating a change in the protocol used in the portal. Are these changes planned to be incorporated in the Home Assistant Integration?
This integration is Suuper!
/JanneS

**Comment by Thomas55555** [comment]
Yes, it's planned. But I'm not sure, if I can test it, because it's winter here.

**Comment by Thomas55555** [comment]
> Hi I’ve got an email from Husquvarna stating a change in the protocol used in the portal. Are these changes planned to be incorporated in the Home Assistant Integration? This integration is Suuper! /JanneS

Okay, I've made the PR ready: https://github.com/Thomas55555/aioautomower/pull/543
Do you have the possibility to tests in on December 20th?

**Reply by JanneSaare** [reply]
Hi!
Great... 🥇  But sorry... Winter Sweden... Cold... Same problem as you have...
